Apps/Src/ReadCarCAN.c

```diff
@@ -119,13 +119,13 @@ static inline void chargingEnable(void) {
  * 
 */
 static void arrayRestart(void *p_tmr, void *p_arg){
-
+    Minion_Error_t Merr;
     if(regenEnable){    // If regen has been disabled during precharge, we don't want to turn on the main contactor immediately after
-        Contactors_Set(ARRAY_CONTACTOR, ON, false);
+        Contactors_Set(ARRAY_CONTACTOR, (true & Minion_Read_Input(IGN_1, &Merr)), false); //turn on array contactor if the ign switch lets us
```

Comment:
Why are you bitwise anding a boolean with true? Just use `Minion_Read_Input` there.

Reply:
lolzies